xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -471,27 +546,33 @@ require(['jquery', 'xwiki-meta', 'xwiki-l10n!xwiki-like-messages'], function ($,
   #set ($optLikeCount = $services.like.getLikes($doc))
   #if ($optLikeCount.isPresent())
     #set ($likeCount = $optLikeCount.get())
-    #set ($discard = $xwiki.ssx.use('XWiki.Like.LikeUIX'))
+    #set ($discard = $xwiki.ssx.use('XWiki.Like.LikeUIX', {'theme': $themeDocFullName}))
     #set ($discard = $xwiki.jsx.use('XWiki.Like.LikeUIX'))
     &lt;div class="like-container"&gt;
       &lt;input id="is-liked" type="hidden" value="$services.like.isLiked($doc)" /&gt;
       #if ($isGuest || $services.like.isLiked($doc))
-        #set ($btnClass = "btn-default")
+        #set ($btnClass = "btn-primary")
+        #set ($heartIconName = 'heart')
         #set ($titleMessageTranslationKey = "like.button.title.unlike")
       #elseif (!$isGuest)
-        #set ($btnClass = "btn-primary")
+        #set ($btnClass = "btn-default")
+        #set ($heartIconName = 'heart_empty')
         #set ($titleMessageTranslationKey = "like.button.title.like")
       #end
       #set ($disabledButton = false)
       #if (!$services.like.isAuthorized($doc))
         #set ($disabledButton = true)
         #set ($titleMessageTranslationKey = "like.button.title")
       #end
-      &lt;button class="like-button btn $btnClass badge"
+      #set($likersText = $services.localization.render("like.likers.link.many", [$likeCount]))
+      #if ($likeCount == 0)
+        #set($likersText = $services.localization.render("like.likers.link.zero", [$likeCount]))
+      #elseif ($likeCount == 1)
+        #set($likersText = $services.localization.render("like.likers.link.one", [$likeCount]))
+      #end
+      &lt;button class="like-button btn $btnClass"
         #if($disabledButton) disabled #{end}
-        title="$escapetool.xml($services.localization.render($titleMessageTranslationKey, [$likeCount]))"&gt;
-          $services.icon.renderHTML('heart') &lt;span class="like-number"&gt;$likeCount&lt;/span&gt;
-      &lt;/button&gt;
+        title="$escapetool.xml($services.localization.render($titleMessageTranslationKey, [$likeCount]))"&gt;$services.icon.renderHTML($heartIconName)&lt;/button&gt; &lt;span class="like-number"&gt;&lt;a href="$escapetool.xml($doc.getURL('view', 'viewer=likers'))" title="$escapetool.xml($services.localization.render('like.likers.link.title', [$likeCount]))"&gt;$escapetool.xml($likersText)&lt;/a&gt;&lt;/span&gt;
```

Comment:
Here you pass the ``$likeCount`` parameter even if it's not used (by the new version of the translation message).

Also, this line is too long (would be nice to limit the line length to 120 chars).